lib/dialects/mssql/index.js

```diff
@@ -3,9 +3,47 @@
 var BaseDialect = require('../base');
 var _ = require('underscore');
 var util = require('util');
+var templatesInit = require('./templates');
+var blocksInit = require('./blocks');
 
 var Dialect = module.exports = function(builder) {
+
+	builder._pushValue = function(value) {
```

Comment:
You shouldn't override `builder._pushValue`, you should override `value` block and do your specific stuff before parent `value` block call. See example in postgresql dialect: https://github.com/2do2go/json-sql/blob/master/lib/dialects/postgresql/blocks.js#L6